dashboard/test/ui/features/hamburger.feature

```diff
@@ -11,27 +11,26 @@ Feature: Hamburger dropdown
     Given I create a student named "Sally Student"
     Then I wait until I am on "http://studio.code.org/home"
     Then I wait to see "#hamburger-icon"
+    Then I wait to see "#help-button"
     And I click selector "#hamburger-icon"
     Then I wait to see "#hamburger-contents"
-    And I see "#report-bug"
-    And I see "#support"
-    And I see ".divider#before-pegasus"
     And I see "#learn"
     And I see "#stats"
     And I see "#help-us"
     And I see "#about_entries"
     And I see "#educate_entries"
+    Then I click selector "#help-button"
+    Then I wait to see "#help-contents"
+    And I see "#report-bug"
+    And I see "#support"
```

Comment:
could we either break each of these tests up into two separate tests, or rename them to make it clear they're testing two separate things?

Reply:
I renamed the tests. Is that what you were saying or should I also rename this feature file? I'm hesitant to separate the tests because these are testing the same page and it seems unnecessary to load the test separately for two elements.

Reply:
oh, I was referring to updating the tests as you did but yeah, it looks like the feature file could use renaming, too!